File: 2026POTD/JANUARY/02-01-2026/961.c
int repeatedNTimes(int* nums, int numsSize) {
    int freq[10001] = {0};
    for (int i = 0; i < numsSize; i++) {
        if (++freq[nums[i]] > 1) return nums[i];
    }
    return -1;
}

File: 2026POTD/JANUARY/02-01-2026/961.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var repeatedNTimes = function(nums) {
    const set = new Set();
    for (const x of nums) {
        if (set.has(x)) return x;
        set.add(x);
    }
};